server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java

```diff
@@ -670,6 +687,8 @@ public void update(LiveTServerSet current, Set<TServerInstance> deleted,
 
         final long actionComplete = System.nanoTime();
 
+        gcRunMetrics.setPostOpDuration(actionComplete - actionStart);
```

Comment:
Hopefully, the API understands this is in nanos?

Reply:
The conversion to millis is performed when the metrics are gathered / reported - will modify the naming so that it is clear that it expects nanoseconds.